apps/next-evans/components/pages/home/news-feed/news-feed.ts

```diff
@@ -20,16 +20,16 @@ export interface FeedItem {
 }
 
 type NewsfeedDocumentProps = Pick<
-  DocumentWithMeta,
-  | 'authorName'
-  | 'dateAdded'
+  Document,
   | 'id'
-  | 'title'
-  | 'authorSlug'
   | 'slug'
+  | 'title'
   | 'editions'
-  | 'mostModernEdition'
```

Comment:
the removed lines here and above came from me being a bit paranoid about whether our types were truly accurate, so in just about every component, i did an experiment of commenting out various things we Pick, and seeing if it still compiled, plus some manual review. i was able to find a decent number of cases where we were passing unused props.

going forward, i want to be pretty disciplined about describing our types carefully and not allowing extra props through.